aws/resource_aws_s3_bucket.go

```diff
@@ -76,6 +76,14 @@ func resourceAwsS3Bucket() *schema.Resource {
 				Default:       "private",
 				Optional:      true,
 				ConflictsWith: []string{"grant"},
+				ValidateFunc: validation.StringInSlice([]string{
+					s3.BucketCannedACLPrivate,
+					s3.BucketCannedACLPublicRead,
+					s3.BucketCannedACLPublicReadWrite,
+					s3.BucketCannedACLAWSExecRead,
+					s3.BucketCannedACLAuthenticatedRead,
+					s3.BucketCannedACLLogDeliveryWrite,
```

Comment:
This doesn't seem to exist?

Reply:
![image](https://user-images.githubusercontent.com/8305363/94368969-51168900-00a4-11eb-8d08-07e47176fe05.png)

Reply:
as in the other comment, yeah you're correct